src/utilities/types.ts

```diff
@@ -152,4 +191,15 @@ export const TYPES: { [key: string]: Partial<IFieldConfig> } = {
     },
     render: stripFieldConfig(parseAndPreserveNewlines),
   },
+  url: {
+    editComponent: Antd.Input,
+    editProps: { type: 'url' },
+    formValidationRules: {
+      url: {
+        message: 'Not a valid website (URLs should start with http:// or https://)',
+        type: 'url',
+      },
+    },
+    render: stripFieldConfig(formatWebsite),
```

Comment:
A rename of this function should probably be on our radar. I assume it was originally meant to do just that - strip an optional field config parameter - but now it's being used to strip all sorts of optional parameters and I have to look up what it does every time I review 😆

Reply:
Na, it's being used to strip the second param here, the fieldConfig param. What name would be more clear?

Reply:
Something like `stripOptionalParameters` or `stripArguments`, as this function does not deal with fieldConfigs specifically